app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class RoomProfilePage extends ConsumerWidget {
+  const RoomProfilePage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final room = ref.watch(currentConvoProvider)!;
+    final members =
+        ref.watch(chatMembersProvider(room.getRoomIdStr())).requireValue;
+    final myMembership =
+        ref.watch(spaceMembershipProvider(room.getRoomIdStr()));
+    final List<Widget> topMenu = [
+      Text(
+        'Members (${members.length})',
+        style: Theme.of(context).textTheme.titleMedium,
+      ),
+    ];
+    if (myMembership.hasValue) {
+      final membership = myMembership.value!;
+      if (membership.canString('CanInvite')) {
+        topMenu.add(
+          IconButton(
+            icon: Icon(
+              Atlas.plus_circle_thin,
+              color: Theme.of(context).colorScheme.neutral5,
+            ),
+            iconSize: 28,
+            color: Theme.of(context).colorScheme.surface,
+            onPressed: () => context.pushNamed(
+              Routes.spaceInvite.name,
+              pathParameters: {'spaceId': room.getRoomIdStr()},
+            ),
+          ),
+        );
+      }
+    }
+
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: [
+          SliverAppBar(
+            elevation: 0.0,
+            actions: <Widget>[
+              PopupMenuButton<int>(
+                icon: const Icon(Atlas.dots_vertical, size: 22),
+                itemBuilder: (context) => [
+                  PopupMenuItem<int>(
+                    value: 0,
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        const Text('Leave Room'),
+                        const SizedBox(width: 50),
+                        Icon(
+                          Icons.exit_to_app_outlined,
+                          color: Theme.of(context).colorScheme.errorContainer,
+                        )
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+            ],
+          ),
+          SliverToBoxAdapter(
+            child: Consumer(
+              builder: (context, ref, child) {
+                final convoProfile = ref.watch(chatProfileDataProvider(room));
+                final roomId = room.getRoomIdStr();
+                return convoProfile.when(
+                  data: (profile) => Column(
+                    children: [
+                      Center(
+                        child: Padding(
+                          padding: const EdgeInsets.only(
+                            top: 38,
+                            bottom: 12,
+                          ),
+                          child: SpaceParentBadge(
+                            badgeSize: 20,
```

Comment:
if the avatar is larger (75), we probably want the badge to be larger, too, no? (like 40 sounds about right)?